packages/webdriverio/src/commands/browser/debug.ts

```diff
@@ -33,14 +32,14 @@
 import { serializeError } from 'serialize-error'
 import WDIORepl from '@wdio/repl'
 
-export default function debug(commandTimeout = 5000) {
+export default function debug(this: WebdriverIO.BrowserObject, commandTimeout = 5000) {
     const repl = new WDIORepl()
     const { introMessage } = WDIORepl
 
     /**
      * run repl in standalone mode
      */
-    if (!process.env.WDIO_WORKER) {
+    if (!process.env.WDIO_WORKER || typeof process.send !== 'function') {
```

Comment:
is it intentional? Didn't quite get the logic

Reply:
`process.send` is not available in parent processes which got recognised by TS so I had to add this little check.